sechub-pds-solutions/gitleaks/docker/Gitleaks.dockerfile

```diff
@@ -5,7 +5,7 @@ ARG BASE_IMAGE
 
 FROM ${BASE_IMAGE}
 
-ARG GITLEAKS_VERSION=8.16.4
+ARG GITLEAKS_VERSION=8.18.2
```

Comment:
This is just a fallback.
In the image build the real value is taken from `sechub-pds-solutions/gitleaks/env`.
There the version is already 8.18.2
We could think about leaving it empty here(?)